setup.py

```diff
@@ -15,9 +15,7 @@
 
 install_requires = [
     'Click>=6.0',
-    # We need the functionality provided by this branch. See this PR for more info:
-    # https://github.com/cookiecutter/cookiecutter/pull/907
-    'cookiecutter @ https://github.com/zillow/cookiecutter/tarball/strip_dir',
```

Comment:
It was this line that was preventing `pip install battenberg` working as `pip` checks that all dependencies are also from PyPI:

```
 alexla@ZG01380  ~/Code/amenity-val   master  pip install battenberg
Collecting battenberg
  Downloading https://files.pythonhosted.org/packages/01/5e/bf1e3917fb609f1587129eecae8aaafb497a3c95a95ed99a84d3eaea86e9/battenberg-0.1.0.tar.gz
ERROR: Packages installed from PyPI cannot depend on packages which are not also hosted on PyPI.
battenberg depends on cookiecutter@ https://github.com/zillow/cookiecutter/tarball/strip_dir
```

This entire PR is just to remove this forked dependency :D